common/static/common/js/components/views/feedback_move.js

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
Since `common/static/common/js/components/views/` only contains reusable components, can we move this file to some other appropriate location and change it's name to something `undo_move_xblock`.